static/app/components/forms/utils/getAllowAutosize.ts

```diff
@@ -0,0 +1,3 @@
+export default function getAllowAutosize() {
```

Comment:
damn, is there a different way we can mock this?

At minimum let's have a comment here as to why there's just a function that always returns true lol

Reply:
Yeah I'll add a comment before merging